apps/public-docsite-v9/.storybook/manager-head.html

```diff
@@ -42,12 +42,16 @@
 
   > 💡 NOTE:
   >
-  > This is brittle way for providing custom non thenable styles for manager UI
+  > This is a brittle way for providing custom non thenable styles for manager UI.
   >
   > Those selectors might change on any storybook version bump.
  -->
 
 <style>
+  /*
+   * Heads Up!
+   * Keep this in sync with the styles in packages/web-components/.storybook/manager-head.html
```

Comment:
why is this needed, nothing like this exists in current master. lets focus only on web-component relate things and not touch react specific domain. please undo these changes.

Reply:
@miroslavstastny to give context here as well